interface/src/instruction.rs

```diff
@@ -2063,6 +2110,37 @@ pub fn withdraw_excess_lamports(
     })
 }
 
+/// Creates an `UnwrapLamports` instruction
+pub fn unwrap_lamports(
+    token_program_id: &Pubkey,
+    source_pubkey: &Pubkey,
+    destination_pubkey: &Pubkey,
+    authority_pubkey: &Pubkey,
+    signer_pubkeys: &[&Pubkey],
+    amount: Option<u64>,
+) -> Result<Instruction, ProgramError> {
+    check_program_account(token_program_id)?;
```

Comment:
Since p-token will support this too, may as well relax the check to either token program:
```suggestion
    check_spl_token_program_account(token_program_id)?;
```